packages/api/emails/survey/html.pug

```diff
@@ -96,12 +100,12 @@ html
 
             hr
 
-            h3 DEADLINE:
+            strong(class="deadline") DEADLINE:
```

Comment:
Just may want to merge in the changes from the other branch before you merge this one. :)